python/cudf/cudf/core/frame.py

```diff
@@ -2688,7 +2688,13 @@ def acos(self):
                     1.5707963267948966,  1.266103672779499],
                     dtype='float64')
         """
-        return self._unaryop("acos")
+        if self.iloc[0].dtype == "int":
```

Comment:
```suggestion
        if self.iloc[0].dtype.kind in ("i", "u"):
```
This check would fail when the dtype is an unsigned int, for example:
```
np.dtype("uint32") == "int"
False
```

Reply:
This looks good to me! I'll add this commit and test it out for sure tomorrow